sorc/rap_subflds_g2.fd/MAKEPRECIP.f90

```diff
@@ -52,6 +53,22 @@ PROGRAM MAKEPRECIP
       REAL,     ALLOCATABLE :: APCPOUT(:),CAPCPOUT(:),SNOWOUT(:)
       REAL,     ALLOCATABLE :: GRAUPELOUT(:)
       LOGICAL,  ALLOCATABLE :: MASK(:)
+
+      real :: fhr3, fhr4
+      integer :: lugb2, lugi2, lugb5, lugb6, lugb7, lugb8, lugb9
+      integer :: istat, numval, iretgb, kret, iret_early, igdnum, ihrs1
+
+      external baopenr
+      external baopen
+      external baclose
+      
+      external getgb2
+      external putgb2
+
+      igdnum = 0
+      fhr3 = 0.0
+      fhr4 = 0.0
+      ihrs1 = 0
```

Comment:
I initialized these previously uninitialized values to 0, perhaps there's a better value for some of them

Reply:
Current operation set 0 to those uninitialized variable. It is good to set them 0 here.